tools/image_processing/bia-ftplinks/biaftplink.xml

```diff
@@ -20,8 +20,10 @@
         <param name="ftplink_output" type="boolean" label="Generate FTP links?" help="If set, a file containing FTP links associated with the accession will be generated." />
     </inputs>
     <outputs>
-        <collection name="images" type="list" label="${tool.name} on ${on_string}: TIFF Images">
+        <collection name="images" type="list" label="${tool.name} on ${on_string}: Images">
             <discover_datasets pattern="(?P&lt;designation&gt;.*)\.tif" format="tif" directory="ftp.ebi.ac.uk" recurse="true" />
+            <discover_datasets pattern="(?P&lt;designation&gt;.*)\.png" format="png" directory="ftp.ebi.ac.uk" recurse="true" />
```

Comment:
should this 3 different collections? what is the usecase? Or should the users then filter the collection afterwards according to filetypes?

Reply:
in most cases, there is only one type of images. But it is better to create different collections.